sdk/servicebus/service-bus/test/common/testUtils.ts

```diff
@@ -0,0 +1,583 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import chai from "chai";
+import {
+  SendableMessageInfo,
+  QueueClient,
+  TopicClient,
+  ServiceBusClient,
+  SubscriptionClient,
+  delay,
+  ReceiveMode,
+  ServiceBusMessage
+} from "../../src";
+import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
+import { ServiceBusManagementClient } from "@azure/arm-servicebus";
+import { SBQueue, SBTopic, SBSubscription } from "@azure/arm-servicebus/esm/models";
+import { Constants, getEnvVars } from "./environmentVariables";
```

Comment:
CI is failing as `Constants` doesn't exist anymore

Reply:
Yes, please ignore CI for now.
Will be adding in other updated files shortly.